lists/global.csv

```diff
@@ -1234,3 +1234,10 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
 https://mitm.watch/,ANON,Anonymization and circumvention tools,2019-03-20,magma,This site detects encryption-removing MITM using logic from NDSS17 interception paper
 https://credibilitycoalition.org/,CULTR,Culture,2019-04-19,OONI,Reportedly inaccessible on WE in Egypt (but probably worth testing globally)
 https://black.riseup.net/,ANON,Anonymization and circumvention tools,2019-04-26,LEAP,
+https://gitlab.com/,HOST,Hosting and Blogging Platforms,2019-07-17,chelchela,
+https://termux.com/,HOST,Hosting and Blogging Platforms,2019-07-17,chelchela,
+https://dl.bintray.com/termux/,HOST,Hosting and Blogging Platforms,2019-07-17,chelchela,
```

Comment:
Why termux specifically and not the root of bintray.com?

Reply:
At the first start of the Termux app, it must communicate with its site, otherwise the application will remain in loading mode. To be sure, I'll re-analyze the app.

Reply:
I made a mistake
The termux.org and termux.net addresses are used :
https://github.com/termux/termux-app/blob/370ac2bd8911b5632933975300e779f722ad03af/app/src/main/java/com/termux/app/TermuxInstaller.java#L174-L175

The following URLs should be tested for proper analysis:
https://termux.org/bootstrap-arm.zip
https://termux.net/bootstrap/bootstrap-arm.zip

Reply:
Just for having status code: 200 at dl.bintary.com which I think is more important than bintray.com.

Reply:
If that is the goal, perhaps we can use the more "neutral" URL for bintray: https://dl.bintray.com/bintray/.

Is this in response to some incident of blocking of bintray.com?

Reply:
Does this URL respond to you?!

Reply:
Yeah you are right it doesn't respond reliably. How about https://dl.bintray.com/jfrog/xray/?

Reply:
From the Netherlands:
![image](https://user-images.githubusercontent.com/12384263/62563188-24ddee80-b88b-11e9-9d30-d53983aeb323.png)

Reply:
Yeah you are right, the `/bintray` URL does not work well.

What about `/jfrog/xray` (jfrog is the company behind bintray)?

Reply:
done